packages/orderbook/test/js_api/gui.test.ts

```
@@ -994,7 +1005,9 @@ ${dotrainWithoutVaultIds}
 			assert.equal(deployment.deployment.order.inputs[0].vaultId, undefined);
 			assert.equal(deployment.deployment.order.outputs[0].vaultId, undefined);
 
-			gui = await DotrainOrderGui.deserializeState(testDotrain, gui.serializeState());
+			let serializedState = extractWasmEncodedData<string>(gui.serializeState());
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'serializedState' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)